frontend/src/styles/Explanation.css

```diff
@@ -0,0 +1,13 @@
+.jumbotron
+{
+    background-color: #a6e2dc !important;
+    width: 95vw;
+    height: 55vh;
+    margin: 0 auto;
+    align-self: center;
+}
+
+.my-4
```

Comment:
My suggestion here is to give a better name to this class. The classname alone doesnt tell me much about what it does.